src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.Helix.targets

```diff
@@ -84,7 +86,10 @@
     <HelixScriptRoot>%HELIX_SCRIPT_ROOT%\</HelixScriptRoot>
     <RunnerScript>%HELIX_CORRELATION_PAYLOAD%\RunnerScripts\scriptrunner\scriptrunner.py</RunnerScript>
   </PropertyGroup>
-
+  <PropertyGroup Condition="'$(TargetsWindows)' == 'true' AND '$(UseNativeCompileRunner)' == 'true' ">
+    <RunnerScript>%HELIX_CORRELATION_PAYLOAD%\RunnerScripts\scriptrunner\nativecompilerunner.py --secondary_queue $(SecondaryQueue) --secondary_payload_dir %HELIX_WORKITEM_PAYLOAD%\native </RunnerScript>
```

Comment:
Don't you think this is baking in a little too much knowledge of our native builds into the common helix scripts/targets?

For example it is always passing the "native" directory.